app/src/main/java/cm/aptoide/pt/app/AppViewAnalytics.java

```diff
@@ -220,6 +221,21 @@ public void sendAppcInfoInteractEvent() {
         AnalyticsManager.Action.CLICK, getViewName(true));
   }
 
+  public void sendDonateClickAfterInstall() {
+    analyticsManager.logEvent(createMapData(ACTION, "Donations after install view"),
```

Comment:
According to the ticket this should be donate_click_after_install

Reply:
fixed